src/pluto/format/webp.cr

```diff
@@ -16,10 +16,11 @@ module Pluto::Format::WebP
       alpha = Array.new(width * height) { 0u8 }
 
       (width * height).times do |index|
-        red.unsafe_put(index, buffer[index * 4])
-        green.unsafe_put(index, buffer[index * 4 + 1])
-        blue.unsafe_put(index, buffer[index * 4 + 2])
-        alpha.unsafe_put(index, buffer[index * 4 + 3])
+        position = index * 4
```

Comment:
Honestly not sure with modern machines - is multiplication the same perf as a bitshift? i.e. this vs `index << 2`

Reply:
#### Before

|                            | Time     | Memory  |
| -------------------------- | -------- | ------- |
| bilinear_resize!(640, 480) | 6.76ms   | 1.17MiB |
| brightness!(1.4)           | 1.84ms   | 0B      |
| box_blur!(10)              | 7.94ms   | 600kiB  |
| channel_swap!(:red, :blue) | 63.0ns   | 0B      |
| contrast!(128)             | 2.07ms   | 0B      |
| gaussian_blur!(10)         | 23.05ms  | 1.75MiB |
| horizontal_blur!(10)       | 3.31ms   | 300kiB  |
| vertical_blur!(10)         | 4.29ms   | 300kiB  |
| crop!(200, 200, 100, 100)  | 43.6µs   | 102kiB  |
| to_jpeg(IO::Memory.new)    | 3.77ms   | 964kiB  |
| to_png(IO::Memory.new)     | 37.28ms  | 1.3MiB  |
| to_ppm(IO::Memory.new)     | 2.39ms   | 2.0MiB  |
| to_webp(IO::Memory.new)    | 106.06ms | 1.23MiB |

#### After

|                            | Time     | Memory  |
| -------------------------- | -------- | ------- |
| bilinear_resize!(640, 480) | 6.16ms   | 1.17MiB |
| brightness!(1.4)           | 1.8ms    | 0B      |
| box_blur!(10)              | 8.55ms   | 600kiB  |
| channel_swap!(:red, :blue) | 70.0ns   | 0B      |
| contrast!(128)             | 2.03ms   | 0B      |
| gaussian_blur!(10)         | 23.07ms  | 1.75MiB |
| horizontal_blur!(10)       | 3.32ms   | 300kiB  |
| vertical_blur!(10)         | 4.33ms   | 300kiB  |
| crop!(200, 200, 100, 100)  | 46.79µs  | 102kiB  |
| to_jpeg(IO::Memory.new)    | 3.79ms   | 964kiB  |
| to_png(IO::Memory.new)     | 37.08ms  | 1.3MiB  |
| to_ppm(IO::Memory.new)     | 2.39ms   | 2.0MiB  |
| to_webp(IO::Memory.new)    | 105.87ms | 1.23MiB |

Reply:
Looks like we can leave things as is?